salt/_pillar/metalk8s.py

```diff
@@ -37,9 +39,30 @@ def _load_networks(config_data):
     }
 
 
+def _get_labels(config="/etc/kubernetes/admin.conf", context="kubernetes-admin@kubernetes"):
+    # TODO: Use the "bootstrap_config" instead of /etc/kubernetes/admin.conf
```

Comment:
Do we have an debt issue about that? If yes, you can refer to it in the TODO (and if not, we should create one, otherwise this TODO will probably stay here forever).

Reply:
Don't know if it's a real debt. It's just that one day we will likely generate a certs for the salt-master to access apiserver then we will need to use this certs (don't know if the ticket exists)
And btw I think it's not `bootstrap_config`.

I will update the TODO and add ticket number